packages/lumos/manifest.yml

```
@@ -0,0 +1,55 @@
+format_version: 3.1.2
+name: lumos
+title: "Lumos"
+version: 0.1.0
+description: "An integration with Lumos to ship your Activity logs to your Elastic instance."
+type: integration
+categories:
+  - security
+conditions:
+  kibana:
+    version: "^8.12.1"
+  elastic:
+    subscription: "basic"
+screenshots:
+  - src: /img/lumos-screenshot.png
+    title: Lumos screenshot 
+    size: 600x600
+    type: image/png
+  - src: /img/activity-log-screenshot.png
+    title: Activity Log screenshot 
+    size: 1496x433 
+    type: image/png
+icons:
+  - src: /img/lumos-logo.svg
+    title: Lumos logo
+    size: 32x32
+    type: image/svg+xml
+policy_templates:
+  - name: lumos
+    title: Lumos Activity Logs
+    description: Collect Activity Logs from Lumos
+    inputs:
+      - type: httpjson
+        title: "Collect Activity Logs via API"
+        description: "Collecting Activity Logs from Slack via API"
+        vars:
+          - name: api_url
+            type: text
+            title: API URL.
+            description: The root URL for the API endpoints
+            multi: false
+            required: true
+            show_user: false
+            default: https://api.lumos.com
+          - name: http_client_timeout
+            type: text
+            title: HTTP Client Timeout
+            description: Duration before declaring that the HTTP client connection has timed out. Valid time units are ns, us, ms, s, m, h.
+            multi: false
+            required: false
+            show_user: true
+            default: 60s
+owner:
+  github: teamlumos/elastic-integrations
```

```suggestion
  github: elastic/security-service-integrations
```
And we will also need a line added to the .github/CODEOWNERS file with the same team.